CRM/Member/Form/Membership.php

```diff
@@ -1225,7 +1218,7 @@ public function submit() {
 
       if (empty($formValues['source'])) {
         $params['contribution_source'] = ts('%1 Membership: Offline signup (by %2)', [
-          1 => $membershipType,
+          1 =>  $this->getSelectedMembershipLabels(),
```

Comment:
Jenkins doesn't like your style here @eileenmcnaughton `Expected 1 space after "=>"; 2 found`

Reply:
jenkins and I gotta talk